lib/preprocessor.py

```diff
@@ -211,6 +211,7 @@ def server_create(self, ctype, sname, opts):
         if 'rpl_master' in opts:
             temp.rpl_master = self.servers[opts['rpl_master']]
         temp.vardir = self.suite_ini['vardir']
+        temp.use_unix_sockets = self.suite_ini.get('use_unix_sockets', False)
```

Comment:
If ini contains `use_unix_sockets = False` the flag will be `True`

Reply:
Thanks, I”ll fix it.